pkgs/applications/window-managers/i3/pystatus.nix

```diff
@@ -1,31 +1,45 @@
-{ stdenv, fetchFromGitHub, libpulseaudio, python3Packages, extraLibs ? [] }:
+{ stdenv
+, fetchFromGitHub
+, libpulseaudio
+, libnotify
+, gobject-introspection
+, python3Packages
+, wrapGAppsHook
+, extraLibs ? [] }:
 
 python3Packages.buildPythonApplication rec {
   # i3pystatus moved to rolling release:
   # https://github.com/enkore/i3pystatus/issues/584
-  version = "unstable-2019-06-10";
+  version = "unstable-2020-06-12";
   pname = "i3pystatus";
 
-  src = fetchFromGitHub
-  {
+  src = fetchFromGitHub {
     owner = "enkore";
     repo = "i3pystatus";
-    rev = "56ce08d0ff8d5d64950d6b588ebede35a95e0ce2";
-    sha256 = "12938860jbcly1xwhd71jvy2dff28pwv9kqh6mab1859148bzmcg";
+    rev = "dad5eb0c5c8a2ecd20c37ade4732586c6e53f44b";
+    sha256 = "18ygvkl92yr69kxsym57k1mc90asdxpz4b943i61qr0s4fc5n4mq";
   };
 
-  propagatedBuildInputs = with python3Packages; [ keyring colour netifaces psutil basiciw ] ++
-    [ libpulseaudio ] ++ extraLibs;
+  buildInputs = [ libpulseaudio libnotify gobject-introspection ];
```

Comment:
`libpulseaudio` goes to `LD_LIBRARY_PATH`, `gobject-introspection` to `GI_TYPELIB_PATH`. But what about `libnotify`? Are you sure that it is not needed for propagation at runtime?

Reply:
`libnotify` is also in in `GI_TYPELIB_PATH`:
```
$ grep libnotify /nix/store/0cvngq5xlihgmal336xs784pmr6sdirx-i3pystatus-unstable-2020-06-12/bin/i3pystatus-python-interpreter
export GI_TYPELIB_PATH='/nix/store/lwzdlszgqahazib2i1zjkqdf8wfjx820-gdk-pixbuf-2.40.0/lib/girepository-1.0:/nix/store/gsjhvi2nbpbfkf6dcz6ibkl4pn3ksym5-libnotify-0.7.9/lib/girepository-1.0:/nix/store/d6sbxgycqpabzf87pymvk2s0d4dlzbqn-gobject-introspection-1.64.1/lib/girepository-1.0'
```